### ISSUE TITLE
[BUG]: Multiple windows should share login

### ISSUE THREAD

**Issue by jtran** [KittyCAD/modeling-app#3897]
### Describe the bug

Different windows don't share login.

### Steps to Reproduce

Run the app, login, and open a kcl file. Then in a terminal, while the first window is still open, run `pathToAppImage some_file.kcl`. It opens a new window, but it prompts me to log in.

### Expected Behavior

I expected that logging in on one window makes me logged in for all windows.

### Desktop OS

Debian 13

### Version

v0.25.2 release candidate

**Comment by jessfraz** [comment]
thats weird id expect that too since arent they all reading from the same token.txt file?

**Comment by jessfraz** [comment]
I swear i tested this with clicking in finder, maybe something regressed?

**Comment by jessfraz** [comment]
also be sure you dont have any build binaries laying about, and that when you run from cli you have no NODE_ENV set

**Comment by jtran** [comment]
It was the first time I installed the app on this machine's Linux partition. So there weren't older binaries around.

**Comment by jessfraz** [comment]
something must have regressed this is the same bug i literally did a demo of this https://github.com/KittyCAD/modeling-app/issues/3955

**Comment by jessfraz** [comment]
yeah here is the video, this used to work https://twitter.com/jessfraz/status/1831767148316021236

**Comment by jessfraz** [comment]
the auth code freaks me out https://github.com/KittyCAD/modeling-app/issues/3973

**Comment by franknoirot** [comment]
Hey @maxammann let me know if you find anything here; I think this is either similar or duplicate to #3955 which I just closed because [the behavior reported doesn't occur anymore](https://github.com/KittyCAD/modeling-app/issues/3955#issuecomment-2695224314). I still think that the refactoring and cleanup noted by @jessfraz in #3973 still needs to happen, but this specific bug might be resolved?

**Comment by franknoirot** [comment]
Sorry I didn't mean to close it, I wanted to leave it to you! Just adding context

**Comment by maxammann** [comment]
I can not really reproduce this. I deleted all zoo data and then:

1. Start Zoo
2. Login to Zoo
3. Open another window
4. I'm logged in.

**Comment by maxammann** [comment]
So I'd say lets close this. This one hits me though: https://github.com/KittyCAD/modeling-app/issues/5640

**Comment by jtran** [comment]
For this issue, I wonder if this changed it:

- https://github.com/KittyCAD/modeling-app/pull/5377

**Comment by jtran** [comment]
Seems related to:

- https://github.com/KittyCAD/modeling-app/issues/3955
